src/script/api/script_window.hpp

```diff
@@ -2473,6 +2473,7 @@ class ScriptWindow : public ScriptObject {
 	/** Widgets of the #TownAuthorityWindow class. */
 	enum TownAuthorityWidgets {
 		WID_TA_CAPTION                               = ::WID_TA_CAPTION,                               ///< Caption of window.
+		WID_TA_ZONE_BUTTON                           = ::WID_TA_ZONE_BUTTON,                           ///<
```

Comment:
missing comment

Reply:
I noticed it as well, but did not want to push a new commit just because of this one. Nevertheless thanks for pointing it out.